lib/github.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+const async = require('async');
+const Breaker = require('circuit-fuses');
+const schema = require('screwdriver-data-schema');
+const Github = require('github');
+const github = new Github();
+let cached = null;
+
+/**
+ * Github command to run
+ * @param  {Object}   options            An object that tells what command & params to run
```

Comment:
Interesting. I do have that installed but it didn't enforce @method. 
I added them now.